modules/apache-airflow/templates/values.yaml

```diff
@@ -467,7 +469,7 @@ kerberos:
 # Airflow Worker Config
 workers:
   # Number of airflow celery workers in StatefulSet
-  replicas: 2
+  replicas: 1
```

Comment:
Curious why the number of workers got decreased?

Reply:
I only reason I was originally using 2 replicas was because, Airflow support running in master-master mode, meaning database locks in postgres make sure data isn't processed twice. And if one of the replicas goes down it doesn't impact airflow.

The problem though is that redis out-of-the-box does not support multiple replicas so it was the single point of failure.

Since we're now running in on-demand instances instead of spot we're going to get the same reliability I was expecting to get out of 2 replicas.